app.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+const {
+  getTopics
+} = require('./controllers/topics-controllers');
+
+
+
+app.get('/api/topics', getTopics);
+
+app.use((err, request, response, next) =>{
+  if(err.code === '23502'){
+    response.status(400).send({msg: 'Bad request'})
+  }
+  else{
+    next(err)
+  }
+})
+
+app.use((err, request, response, next) =>{
+  if(err.status===404){
+    response.status(404).send()
+  }
+})
+
```

Comment:
You will want a next in your final error handler to pass it to the server which results in a 500 (when all other handlers have failed)